doc/Type/UInt.pod6

```diff
@@ -0,0 +1,12 @@
+=begin pod
+
+=TITLE class UInt
```

Comment:
On Wed, Aug 31, 2016 at 12:40 PM, Zoffix Znet <notifications@github.com> wrote:
>
> In doc/Type/UInt.pod6:
>
> > @@ -0,0 +1,12 @@
> > +=begin pod
> > +
> > +=TITLE class UInt
>
> I meant this:
>
> Elfoonto m: my $int = Int.new: 42;

Later I figured that out.  I threw out a challenge to either correct
the doc or move the text but keep 'UInt' searchable.